polls_plugin/templatetags/polls_filters.py

```diff
@@ -3,6 +3,11 @@
 register = template.Library()
 
 
-@register.filter
-def get_item(dictionary, key):
-    return dictionary.get(str(key), None)
+@register.simple_tag
+def get_vote(user_votes, model_votes, key):
```

Comment:
Consider writing a docstring to describe the returned values. Apparently the second returned value indicates if the " vote" (`assessment` would be better) is an inference or from an actual user. However, isn't this information not already instored in `class Vote`? Why do we need to return it separately Maybe we should add type annotations to `user_votes` to describe what the type of each item is.